code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -243,6 +243,15 @@
 	amount_per_transfer_from_this = 60
 	list_reagents = list(/datum/reagent/medicine/salbutamol = 10, /datum/reagent/medicine/c2/penthrite = 10, /datum/reagent/medicine/oxandrolone = 10, /datum/reagent/medicine/sal_acid = 10 ,/datum/reagent/medicine/omnizine = 10 ,/datum/reagent/medicine/leporazine = 10)
 
+/obj/item/reagent_containers/hypospray/medipen/stimpack/mining //REAL goliath kiting
+	name = "emergency stimpack medipen"
+	desc = "A rapid way to inject some stimulants to move faster while killing local fauna."
+	icon_state = "stimpen"
+	inhand_icon_state = "stimpen"
+	volume = 30
+	amount_per_transfer_from_this = 30
+	list_reagents = list(/datum/reagent/medicine/ephedrine = 8, /datum/reagent/consumable/coffee = 8, /datum/reagent/medicine/epinephrine = 8, /datum/reagent/medicine/morphine = 6)

```

Comment:
this pen is awful awful awful awful for actual fighting on lavaland, and even in PvP, it stinks

Ephedrine can and will make you drop your PKA several times, possibly mid-combat. Coffee and epinephrine will react with each other to form pump-up, a chem with the sole upside of giving you the TRAIT_STUNRESISTANCE trait... which ONLY affects the lengths of knockdowns from stun batons (not any other source of stuns), and is _already provided by the ephedrine anyway_. The epinephrine present in this pen will also react with the penthrite from a luxury survival medipen, causing you to explode if you try to use both of them together (even though this medipen's intended purpose seems to be to be used alongside luxury survival medipens while fighting megafauna). Finally, the morphine in this medipen is present in a large enough dose to guarantee that you'll be knocked out by it (the ephedrine won't even help you with waking up from the chain knockouts after tick 24-ish, because from that point on, morphine will increase your sleeping duration to 4 seconds every 2 seconds, while ephedrine will merely reduce it by 2 seconds every 2 seconds, and the coffee will be long gone by then), which is, of course, a death sentence during a megafauna fight.

In short, literally every chem in this pen actively harms your ability to fight (mega)fauna on lavaland.